WidgetExtension/Common/ArtworkViews.swift

```diff
@@ -2,28 +2,38 @@ import SwiftUI
 
 struct LargeArtworkView: View {
     @State var imageData: Data?
+
+    var showShadow: Bool = true
```

Comment:
Perhaps this one should be `let` ?

Reply:
Curious enough, if I do that the `showShadow` stops being a property in the `struct` constructor. I would have to manually create an `init`, so I think that maybe this is an ok-tradeoff?